.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,39 @@
+---
+name: 🐞 Bug report
+about: Create a bug report
+title: ''
+labels: ["C-bug"]
+assignees: ''
+
+---
+
+<!--
+Note: Please search to see if an issue already exists for the bug you encountered.
+-->
+
+## Environment
+Example:
+- OS: Windows 11
+- CPU: AMD Ryzen 5 5600X
+- Memory: 16GB
+- Storage: SSD 500GB
+- Cuprate: v1.0.0
+
+## Bug
+What is the bug?
+
+### Expected behavior
+What correct beahvior was expected to happen?
+
+## Steps to reproduce
+Example:
+1. In this environment...
+2. With this config...
+3. Run '...'
+4. See error...
+
+## Log
+If possible, add any related logs to help explain the bug.
+
+## Screenshots
+If possible, add screenshots to help explain the bug.
```

Comment:
We should probably add a note here to check the logs for private info that they don't want to make public (IP addresses etc)